internal/generator/generator_test.go

```diff
@@ -4,9 +4,9 @@ import (
 	"bufio"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"net/http"
+	"io"
```

Comment:
Imports must be ordered lexicographically, please put `io` between `fmt` and `log`